server/tests/participant.form.test.js

```diff
@@ -582,4 +582,33 @@ describe('Participants Service', () => {
     expect(query2[0].firstName).toEqual('Eddy');
     expect(query2[0].history.length).toEqual(1);
   });
+
+  it('confirms that reporting works properly', async () => {
```

Comment:
Love this 😍 it even answered a question I was going to ask about how participants hired elsewhere were being handled!

Reply:
Actually looking at the query, I'm wondering if we do need to exclude this particular rejection status from the report.

What I'm thinking is that a participant could be engaged and rejected, and THEN engaged and hired by another employer after the fact. So they would show up in this report with a normal rejection reason (like "withdrew") but they would actually be hired, and therefore NOT a participant that we want to prioritize in the callback process.

The simplest way to remove these participants from the report is just to exclude all participants with a hired status. That will have the knock-on effect of removing the statuses for "hired by other" unless we intentionally include that back into the report, which I don't think we need to do.

So... I think my vote is filter out participants who are hired, which will consequently filter out "hired by other", which is a totally OK outcome for this report (but does mean we need to update the expected output of this test, plus add another one for a normal rejection reason).